Sprint-3/todo-list/script.js

```diff
@@ -1,25 +1,77 @@
-function populateTodoList(todos) {
-  let list = document.getElementById("todo-list");
-  // Write your code to create todo list elements with completed and delete buttons here, all todos should display inside the "todo-list" element.
+//i create a function to populate the todo and display it.
+function populateTodoList() {
+  const todoList = document.getElementById("todo-list");
+  todoList.innerHTML = ""; //to prevent repeating the output in my ul
+  //loop through todos and create the list
+  todos.forEach((todo, index) => {
+    const li = document.createElement("li");
+    const todoText = document.createElement("span");
+    todoText.textContent = todo.task;
+    //apply red color if task is completed
+    if (todo.completed) {
+      todoText.style.textDecoration = "line-through"; //change text decoration to line through if completed
+    }
+    li.appendChild(todoText);
+    const badge = document.createElement("span"); //badge icons
+    badge.classList.add("badge", "bg-primary", "rounded-pill");
+    const checkIcon = document.createElement("i");
+    checkIcon.classList.add("fa", "fa-check"); //check icon
+    checkIcon.setAttribute("aria-hidden", "true");
+    checkIcon.addEventListener("click", () => toggleComplete(index, todoText));
+    badge.appendChild(checkIcon);
+    const trashIcon = document.createElement("i");
+    trashIcon.classList.add("fa", "fa-trash"); //trash icon
+    trashIcon.setAttribute("aria-hidden", "true");
+    trashIcon.addEventListener("click", () => deleteTodo(index));
+    badge.appendChild(trashIcon);
+    li.appendChild(badge);
+    todoList.appendChild(li);
+  });
 }
-
-// These are the same todos that currently display in the HTML
-// You will want to remove the ones in the current HTML after you have created them using JavaScript
 let todos = [
   { task: "Wash the dishes", completed: false },
   { task: "Do the shopping", completed: false },
```

Comment:
this is global variable that you use inside of a function  in the beginning of your script, please move this at the top